lib/Doctrine/Parser/Xml.php

```diff
@@ -121,9 +121,11 @@ public function loadData($path)
      */
     public function prepareData($simpleXml)
     {
+        $children = array();
+        $return = array();
+
         if ($simpleXml instanceof SimpleXMLElement) {
             $children = $simpleXml->children();
-            $return = null;
```

Comment:
This wasn't necessary with `$return` being set as an empty array.

There's an `is_array` check below that would make this return an empty array if `$return` was `null` anyhow.